### PR TITLE
Update icarus configuration

### PR DESCRIPTION
**Pull request by m1cr0man**
Configure new drives

### CODE REVIEW

File: hosts/icarus/hardware-configuration.nix [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-# to /etc/nixos/configuration.nix instead.
-{ config, lib, pkgs, ... }:
+{ config, lib, pkgs, modulesPath, ... }:
 let
   common = import ../../common/variables.nix;
 in {
   imports =
-    [ <nixpkgs/nixos/modules/installer/scan/not-detected.nix>
+    [ (modulesPath + "/installer/scan/not-detected.nix")
     ];
```
Comment (benmcmahon100) [inline hosts/icarus/hardware-configuration.nix:7]: Hangin bracket is a lil weird

**Review by benmcmahon100 — APPROVED** [review]
One small stlye nit

**Review by cmgn — APPROVED** [review]

File: hosts/icarus/hardware-configuration.nix (hunk 1 above)
Comment (m1cr0man) [reply hosts/icarus/hardware-configuration.nix:7]: That's how it's generated. Same for all hardware confs.

Comment (benmcmahon100) [reply hosts/icarus/hardware-configuration.nix:7]: Oh cool

**Pull request merged**